docs/index.rst

```diff
@@ -18,17 +18,9 @@ the xonsh shell
         "Pass the xonsh, Piggy",
         "Piggy glanced nervously into hell and cradled the xonsh",
         "The xonsh is a symbol",
-        "It is pronounced <i>konk</i>",
```

Comment:
It would be nice to leave this one at least, since that is actually how "conch" is pronounced :)